salt/utils/jinja.py

```diff
@@ -98,28 +98,60 @@ def check_cache(self, template):
             self.cached.append(template)
 
     def get_source(self, environment, template):
-        # checks for relative '..' paths
-        if '..' in template:
-            log.warning(
-                'Discarded template path \'%s\', relative paths are '
-                'prohibited', template
-            )
-            raise TemplateNotFound(template)
+        '''
+        Salt-specific loader to find imported jinja files.
+
+        Jinja imports will be interpreted as originating from the top
+        of each of the directories in the searchpath when the template
+        name does not begin with './' or '../'.  When a template name
+        begins with './' or '../' then the import will be relative to
+        the importing file.
+        
+        '''
+        # FIXME: somewhere do seprataor replacement: '\\' => '/'
```

Comment:
Translation of `\` => `/` might be broken.  Help me, Obi Juan, you're my only hope!

Reply:
@dwoz @twangboy When using jinja imports on Windows, are Windows-specific path-separators supported? If so, are they the only ones supported, or are forward slashes also supported? I would assume both are supported, since they are in Python proper, but I don't want to assume without knowing for sure.

Reply:
I use Windows separators in my Jinja. I don't even have to escape them. Jinja apparently does some escaping for you.

Reply:
The ``tpldir = os.path.dirname(template).replace('\\', '/')`` construct was in the original code and appears to do some path translation on Windows.  I'm not sure how to test it to ensure that I haven't broken the functionality.  Is there a unit test?